lib/open_project/open_desk/patches/top_menu_helper_patch.rb

```diff
@@ -35,13 +35,28 @@ def self.included(base)
     module InstanceMethods
       def render_module_top_menu_node(*)
         items = first_level_menu_items_for(:open_desk_menu)
-        unless items.empty?
-          render_menu_dropdown_with_items(
-            label: "",
-            label_options: { icon: "icon-menu", title: I18n.t("open_desk.central_navigation_menu") },
-            items:,
-            options: { drop_down_id: "more-menu", drop_down_class: "drop-down--modules ", menu_item_class: "hidden-for-mobile" }
-          )
+        return if items.empty?
+
+        render Primer::Alpha::Dialog.new(classes: "op-app-menu--item",
+                                         title: I18n.t("open_desk.central_navigation_menu"),
+                                         size: :small,
+                                         menu_id: "op-app-header--modules-menu",
+                                         position: :left) do |dialog|
+          dialog.with_show_button(icon: "op-grid-menu",
+                                  scheme: :invisible,
+                                  classes: "op-app-menu--item-action op-app-header--primer-button",
```

Comment:
The class does not exist any more.
```suggestion
                                  classes: "op-app-header--primer-button",
```